.github/dependabot.yml

```diff
@@ -7,7 +7,7 @@ updates:
     day: wednesday
     time: "03:00"
     timezone: Europe/London
-  open-pull-requests-limit: 5
+  # open-pull-requests-limit: 5
```

Comment:
[open-pull-requests-limit](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit) - The default is 5 so your change wouldnt do anything

Reply:
Hmmm, I didn't know that. I'm fairly sure we used to get spammed with an (apparently) unlimited number.